androidApp/src/main/kotlin/com/tajemniktv/tajsos/MainActivity.kt

```
@@ -307,7 +312,7 @@ class MainActivity : FragmentActivity() {
             }
         try {
             speechRecognizerLauncher.launch(intent)
-        } catch (e: Exception) {
+        } catch (e: android.content.ActivityNotFoundException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)